nrf_security/src/mbedtls/CMakeLists.txt

```diff
@@ -89,87 +146,90 @@ PREPEND(src_x509 ${ARM_MBEDTLS_ROOT}/library/
     x509write_csr.c
 )
 
-PREPEND(src_tls ${ARM_MBEDTLS_ROOT}/library/
+#
+# TLS files
+#
+append_with_prefix(src_tls ${ARM_MBEDTLS_ROOT}/library/
     debug.c
     net_sockets.c
     ssl_cache.c
-    # ssl_ciphersuites.c - replaced in nrf_mbedtls
     ssl_cli.c
     ssl_cookie.c
     ssl_srv.c
     ssl_ticket.c
     ssl_tls.c
 )
 
-# ----------  Replacement sources - modified mbedTLS sources  -------------
+# ----------  Replacement sources - modified mbedTLS sources  -------------<
 
-PREPEND(src_crypto_replacement ${NRF_MBEDTLS_ROOT}/src/mbedtls/replacements/
-    cipher_wrap.c
-    pem.c
+append_with_prefix(src_crypto_replacement
+  ${NRF_MBEDTLS_ROOT}/src/mbedtls/replacements/
+  pem.c
 )
 
-PREPEND(src_tls_replacement ${NRF_MBEDTLS_ROOT}/src/mbedtls/replacements/
-    ssl_ciphersuites.c
+append_with_prefix(src_tls_replacement
+  ${NRF_MBEDTLS_ROOT}/src/mbedtls/replacements/
+  ssl_ciphersuites.c
 )
 
 zephyr_include_directories(${ARM_MBEDTLS_ROOT}/include)
 zephyr_include_directories(${ARM_MBEDTLS_ROOT}/include/mbedtls)
+zephyr_include_directories(${common_includes})
```

Comment:
Isn't this redundant with https://github.com/NordicPlayground/nrfxlib/pull/82/files#diff-b34bd274b90e72abf1c2550f34011a26R19 ?

Reply:
Fixed